### ISSUE TITLE
Click on a displayed tag to search using that tag

### ISSUE THREAD

**Issue by abouffard** [Esri/military-symbol-editor-addin-wpf#57]
Currently, when you select a symbol, you see a display of its tags in an array of blue button-looking shapes.  Currently these are just there as information, but they look like buttons I could click.  I would like to see them actually  become real buttons that, when clicked, would use their tag value to perform another search.

**Comment by nfeuerstein** [comment]
Agreed, or make the tags look like text and not buttons that can be clicked.

**Comment by csmoore** [comment]
We can change the style - the original thought was to make them look like the Pro tags (but these can indeed be clicked to remove the tag - so we could also do that):

![image](https://cloud.githubusercontent.com/assets/3090809/21226556/f8a06ee0-c2a3-11e6-9f44-716b88e6314c.png)
